### ISSUE TITLE
First input after toggling fullscreen not available on MacOS?

### ISSUE THREAD

**Issue by subsoap** [subsoap/defos#117]
https://forum.defold.com/t/first-mouse-click-after-fullscreen-toggle/66076

[fullscreen_mouse.zip](https://github.com/subsoap/defos/files/5092963/fullscreen_mouse.zip)

**Comment by dapetcu21** [comment]
Yeah, I experienced that as well in Interrogation. Not sure why it happens or how to fix it.
